spk/jellyfin/Makefile

```diff
@@ -1,22 +1,24 @@
 # Remember to also update jellyfin-web
 SPK_NAME = jellyfin
-SPK_VERS = 10.8.13
-SPK_REV = 13
+SPK_VERS = 10.9.3
+SPK_REV = 14
 SPK_ICON = src/jellyfin.png
 WIZARDS_DIR = src/wizard/
 DSM_UI_DIR = app
 
-OPTIONAL_DEPENDS = cross/libstdc++
 DEPENDS = cross/jellyfin cross/jellyfin-web
 
+# GLIBC >= 2.28 required
+REQUIRED_MIN_DSM = 7.1
+
 # x64 and armv8 archs are supported only.
 UNSUPPORTED_ARCHS = $(32bit_ARCHS)
 
-MAINTAINER = SynoCommunity
+MAINTAINER = stevenliuit
 DESCRIPTION = "The Free Software Media System. It is an alternative to the proprietary Emby and Plex."
 DISPLAY_NAME = Jellyfin
 STARTABLE = yes
-CHANGELOG = "1. Update jellyfin to 10.8.13<br/>2. Update dotnet to 6.0.25."
+CHANGELOG = "Update jellyfin to 10.9.3<br/>Built with dotnet 8 and nodejs 20."
 HOMEPAGE = https://jellyfin.org
 HELPURL = https://jellyfin.org/docs/general/server/settings.html
 SUPPORTURL = https://jellyfin.org/docs/general/getting-help.html
```

Comment:
Please change 'SPK_DEPENDS = "ffmpeg>=4.2"' to 'SPK_DEPENDS = "ffmpeg6>=6.0" to use an up-to-date ffmpeg version